app/Http/Controllers/API/BeatmapsController.php

```diff
@@ -7,20 +7,62 @@
 
 use App\Models\Beatmap;
 
+/**
+ * @group Beatmaps
+ */
 class BeatmapsController extends Controller
 {
     public function __construct()
     {
         $this->middleware('require-scopes:public');
     }
 
+    /**
+     * Get Beatmap
+     *
+     * Gets beatmap data for the specified beatmap ID.
+     *
+     * ---
+     *
+     * ### Response format
+     *
+     * Returns [Beatmap](#beatmap) object.
+     * Following attributes are included in the response object when applicable,
+     *
+     * Attribute                            | Notes
+     * -------------------------------------|------
+     * beatmapset                           | See [Beatmapset](#beatmapset)
```

Comment:
This should mention `ratings` is included. Link to Beatmapset is probably not needed as it's already linked in Beatmap(Compact) section.